apps/examples/wifi_test/wifi_test.c

```diff
@@ -248,15 +259,16 @@ int main(int argc, FAR char *argv[])
 int wifi_test_main(int argc, char *argv[])
 #endif
 {
-
-	int pid;
-
+#ifdef CONFIG_TASH
+	tash_cmd_install("wifi_scan", wifi_init_task, TASH_EXECMD_SYNC);
+#else
 	/* Create Task For async job */
-	pid = task_create("wifi_test_proc", SCHED_PRIORITY_DEFAULT, 16840, wifi_test_proc, argv);
-	if (pid < 0) {
+	if (task_create("wifi_test_proc", SCHED_PRIORITY_DEFAULT, 16840, wifi_test_proc, argv) < 0) {
```

Comment:
these codes are same with wifi_init_task. just call wifi_init_task for else case.